build.sh

```diff
@@ -90,6 +90,9 @@ function docker_cmd()
         echo "Building $DOCKERHUB_REGISTRY:$HASH_TAG"
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} --file Dockerfile .
 
+        echo "Scanning docker image..."
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL $DOCKERHUB_REGISTRY:${HASH_TAG}
```

Comment:
I think this is also what codacy is complaining about:
```suggestion
        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_REGISTRY":${HASH_TAG}
```

Reply:
Thank you, nice catch!